suites/reef/cephfs/tier-2_cephfs_io.yaml

```diff
@@ -227,11 +235,87 @@ tests:
           Files_per_dir: 1
           File_size: 1k
           Instances: 1
+  - test:
+      abort-on-fail: false
+      desc: "Run VDA Workload"
+      module: cephfs_spec_storage.py
+      name: FS_SPEC_REPL_MAX_MDS_2_VDA
+      polarion-id: "CEPH-XXXXX"
+      config:
+        clients:
+          - client1
+        benchmark: VDA
+        parse_result: True
+        fs_name: cephfs
+        max_mds: 2
+        benchmark_defination:
+          Warmup_time: 30
+          Dir_count: 1
+          Files_per_dir: 1
+          File_size: 1k
+          Instances: 1
+  - test:
+      abort-on-fail: false
+      desc: "Run EDA_BLENDED Workload"
+      module: cephfs_spec_storage.py
+      name: FS_SPEC_REPL_MAX_MDS_2_EDA_BLENDED
+      polarion-id: "CEPH-XXXXX"
+      config:
+        clients:
+          - client1
+        benchmark: EDA_BLENDED
+        parse_result: True
+        fs_name: cephfs
+        max_mds: 2
+        benchmark_defination:
+          Warmup_time: 30
+          Dir_count: 1
+          Files_per_dir: 1
+          File_size: 1k
+          Instances: 1
+  - test:
+      abort-on-fail: false
+      desc: "Run AI_IMAGE Workload"
+      module: cephfs_spec_storage.py
+      name: FS_SPEC_REPL_MAX_MDS_2_AI_IMAGE
+      polarion-id: "CEPH-XXXXX"
```

Comment:
when are we planning to update polarion-id?

Reply:
@AmarnatReddy Please add one polarion testcase for performance test. Other tests can have empty IDs. Lets not create multiple tests just for different workloads provided by SPECStorage.

Reply:
I have created a ploarion id :  CEPH-83588102 and added it to the last test case